.travis.yml

```diff
@@ -3,4 +3,5 @@ rvm:
   - 2.2.1
 
 before_script:
-  RAILS_ENV=test bundle exec rake db:create
+  - RAILS_ENV=test bundle exec rake db:create
+  - bundle exec teaspoon
```

Comment:
Seems strange that this is in the `before_script`. This is part of your actual test script, and not part of setup required for your tests.